app/models/finance/billing_strategy.rb

```diff
@@ -69,6 +69,8 @@ def self.canaries
   def self.daily(options = {})
     raise 'Options must be a hash' unless options.is_a?(Hash)
 
+    Rails.logger.info("Finance::BillingStrategy.daily started for options #{options}")
```

Comment:
maybe remove the log messages in the post- and -pre-pay strategies as that would be a duplication and just noise

Reply:
Well, to be honest, I'd prefer to duplicate and remove later, just in case we miss some important step.

Between this log and `BillingStrategy#daily` there is quite some logic...